shared/js/background/events/3p-tracking-cookie-blocking.js

```diff
@@ -67,7 +77,8 @@ function dropTracking3pCookiesFromRequest (request) {
         } else if (tab && trackerutils.isFirstPartyByEntity(request.url, tab.url)) {
             return { requestHeaders }
         }
-        if (!utils.isCookieExcluded(request.url)) {
+        const cookieFeature = requestIsTracker ? 'trackingCookies3p' : 'nonTracking3pCookies'
+        if (!utils.isCookieExcluded(request.url, cookieFeature)) {
             requestHeaders = requestHeaders.filter(header => header.name.toLowerCase() !== 'cookie')
             devtools.postMessage(request.tabId, 'cookie', {
```

Comment:
Can you add the feature name to the debug message, so we can see which feature dropped the cookie in the debugger?